IDEAS/Buildings/Components/Interfaces/ZoneInterface.mo

```diff
@@ -55,13 +64,24 @@ partial model ZoneInterface "Partial model for thermal building zones"
   Modelica.Blocks.Interfaces.RealInput yOcc if useOccNumInput
     "Control input for number of occupants, used by Occupants.Input and Occupants.AreaWeightedInput"
     annotation (Placement(transformation(extent={{140,20},{100,60}})));
+  Modelica.Blocks.Interfaces.RealInput mWat_flow(unit = "kg/s") 
+    if useWatFlowInput 
+    "Input for injecting water vapor into a zone" annotation(
+    Placement(visible = true, transformation(origin = {0, 50}, extent = {{140, 20}, {100, 60}}, rotation = 0), iconTransformation(origin = {0, 40}, extent = {{140, 20}, {100, 60}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput C_flow 
+    if useCFlowInput 
+    "Input for injecting CO2 into a zone" annotation(
+    Placement(visible = true, transformation(origin = {0, 70}, extent = {{140, 20}, {100, 60}}, rotation = 0), iconTransformation(origin = {0, 80}, extent = {{140, 20}, {100, 60}}, rotation = 0)));  
   Modelica.Blocks.Interfaces.RealInput uLig if useLigCtrInput
     "Lighting control input (1 corresponds to 100%), only used when using LightingControl.Input"
     annotation (Placement(transformation(extent={{140,50},{100,90}}),
         iconTransformation(extent={{-130,-40},{-90,0}})));
-  Modelica.Blocks.Interfaces.RealOutput ppm(unit="1")
+  Modelica.Blocks.Interfaces.RealOutput ppm(unit="1",min=0)
     "CO2 concentration in the zone" annotation (Placement(transformation(extent={{100,-10},
             {120,10}}),           iconTransformation(extent={{100,-10},{120,10}})));
+  Modelica.Blocks.Interfaces.RealOutput phi(unit="1",min=0,max=1)
+    "Relative humidity in the zone [0-1]" annotation (Placement(transformation(extent={{100,0},
+            {120,20}}),           iconTransformation(extent={{100,0},{120,20}})));
```

Comment:
This output connector overlaps with those of the temperature and CO2 concentration (see picture comment 'C_flow'); can we restructure the connectors in a better way?

Reply:
Same comments as above, could use your help on this one.

Reply:
I can replace this one, but in order to fit the output connector of `phi` in line with the other two, I should replace these latter two. Or do I place it at the left side of the component like you did with the input connectors.